onnxruntime/hosting/server_configuration.h

```diff
@@ -9,21 +9,31 @@
 
 #include "boost/program_options.hpp"
 
+#include "core/common/logging/logging.h"
+
 namespace onnxruntime {
 namespace hosting {
 
-enum Result { ExitSuccess = 1,
-              ExitFailure,
-              ContinueSuccess };
+enum class Result { ExitSuccess = 1,
+                    ExitFailure,
+                    ContinueSuccess };
 
 namespace po = boost::program_options;
 
 // Wrapper around Boost program_options and should provide all the functionality for options parsing
 // Provides sane default values
 class ServerConfiguration {
  public:
+  const std::string full_desc = "ONNX Hosting: host an ONNX model for inferencing with ONNXRuntime";
+  std::string model_path;
+  std::string address = "0.0.0.0";
+  int http_port = 8001;
+  int num_http_threads = std::thread::hardware_concurrency();
+  onnxruntime::logging::Severity logging_level{};
```

Comment:
We should set a default one, e.g. warning.

Reply:
We do but the string field is private so it's not here and the enum defaults to 0.